game.rb

```diff
@@ -0,0 +1,96 @@
+# frozen_string_literal: true
+
+# player class
+class Player
+  attr_accessor :name, :hand, :bank, :total, :bet
+
+  def initialize(name, bank_roll)
+    @name = name
+    @hand = []
+    @bank_roll = bank_roll
+    @total = 0
+    @bet = 0
+  end
+
+  def add_card_in_hand(card)
+    card.score = 1 if card.score == 11 && @total > 11
+    @total += card.score
+    @hand << card
+  end
+
+  def intrdouce
+    puts "I am #{@name}, I have $#{@bank_roll} left."
+  end
+end
+
+# card class
+class Card
+  attr_accessor :value, :suit, :score
+
+  def initialize(value, suit, score)
+    @value = value
+    @suit = suit
+    @score = score
+  end
+
+  def show_card
+    puts "#{@value}-#{@suit} (#{@score})"
```

Comment:
Since we've defined `attr_accessor` for these instance variables, can we access them without `@`?

Reply:
I tried accessing these variables without `@` and it worked, feedback incorporated.